### DISCUSSION TITLE
Side Tree bug when selecting folders with subfolders

### DISCUSSION THREAD

**Discussion by jo-chemla** [requarks/wiki#5311] [Error / Bug Report] [answered]
Hi there, 
First of all, thanks for the massive work on wiki.js!
Since a very recent wikijs 2 update (which auto-installed, did not notice which version change produced that bug, but current version is up to date 2.5.282), a new bug spawned in the side tree. We have multiple folders at root level, some with and some without subfolders. 
 - When clicking in the side-tree on folders with only pages as children (and no subfolders), then the side tree presents to the user the content of that root folder `/root / folder-name / list of folder content`. 
 - But when clicking in the tree on folders which do possess sub-folders, then their content is not shown on the side tree. If I click in that order on folder1-folder2-folder3 where folder1 and folder2 have subfolders, and folder3 does not, then the side-tree content is updated and displays in the side-panel `/root / folder1 / folder2 / folder3 / list of folder3 content`.

**Comment by ghost** [comment, marked as answer]
Fixed in #5304

**Comment by peteruithoven** [comment]
Also see: https://github.com/requarks/wiki/discussions/5295
